pandas/core/window/common.py

```diff
@@ -250,31 +250,10 @@ def _get_center_of_mass(comass, span, halflife, alpha):
     return float(comass)
 
 
-def _offset(window, center):
+def _calculate_center_offset(window):
```

Comment:
as a followup typing & doc-strings would be great. also ok to de-privatize these